lib/command.h

```diff
@@ -91,100 +91,101 @@ struct host {
+	BGP_IPV6_NODE,		/* BGP IPv6 address family */
+	BGP_IPV6M_NODE,		/* BGP IPv6 multicast address family. */
+	BGP_IPV6L_NODE,		/* BGP IPv6 labeled unicast address family. */
+	BGP_VRF_POLICY_NODE,    /* BGP VRF policy */
+	BGP_VNC_DEFAULTS_NODE,  /* BGP VNC nve defaults */
+	BGP_VNC_NVE_GROUP_NODE, /* BGP VNC nve group */
+	BGP_VNC_L2_GROUP_NODE,  /* BGP VNC L2 group */
+	RFP_DEFAULTS_NODE,      /* RFP defaults node */
+	BGP_EVPN_NODE,		/* BGP EVPN node. */
+	BGP_SRV6_NODE,		/* BGP SRv6 node. */
+	OSPF_NODE,		/* OSPF protocol mode */
+	OSPF6_NODE,		/* OSPF protocol for IPv6 mode */
+	LDP_NODE,		/* LDP protocol mode */
+	LDP_IPV4_NODE,		/* LDP IPv4 address family */
+	LDP_IPV6_NODE,		/* LDP IPv6 address family */
+	LDP_IPV4_IFACE_NODE,    /* LDP IPv4 Interface */
+	LDP_IPV6_IFACE_NODE,    /* LDP IPv6 Interface */
+	LDP_L2VPN_NODE,		/* LDP L2VPN node */
+	LDP_PSEUDOWIRE_NODE,    /* LDP Pseudowire node */
+	ISIS_NODE,		/* ISIS protocol mode */
+	ACCESS_NODE,		/* Access list node. */
+	PREFIX_NODE,		/* Prefix list node. */
+	ACCESS_IPV6_NODE,       /* Access list node. */
+	ACCESS_MAC_NODE,	/* MAC access list node*/
+	PREFIX_IPV6_NODE,       /* Prefix list node. */
+	AS_LIST_NODE,		/* AS list node. */
+	COMMUNITY_LIST_NODE,    /* Community list node. */
+	COMMUNITY_ALIAS_NODE,   /* Community alias node. */
+	RMAP_NODE,		/* Route map node. */
+	PBRMAP_NODE,		/* PBR map node. */
+	SMUX_NODE,		/* SNMP configuration node. */
+	DUMP_NODE,		/* Packet dump node. */
+	FORWARDING_NODE,	/* IP forwarding node. */
+	PROTOCOL_NODE,		/* protocol filtering node */
+	MPLS_NODE,		/* MPLS config node */
+	PW_NODE,		/* Pseudowire config node */
+	SEGMENT_ROUTING_NODE,   /* Segment routing root node */
+	SR_TRAFFIC_ENG_NODE,    /* SR Traffic Engineering node */
+	SR_SEGMENT_LIST_NODE,   /* SR segment list config node */
+	SR_POLICY_NODE,		/* SR policy config node */
+	SR_CANDIDATE_DYN_NODE,  /* SR dynamic candidate path config node */
+	PCEP_NODE,		/* PCEP node */
+	PCEP_PCE_CONFIG_NODE,   /* PCE shared configuration node */
+	PCEP_PCE_NODE,		/* PCE configuration node */
+	PCEP_PCC_NODE,		/* PCC configuration node */
+	SRV6_NODE,		/* SRv6 node */
+	SRV6_LOCS_NODE,		/* SRv6 locators node */
+	SRV6_LOC_NODE,		/* SRv6 locator node */
+	VTY_NODE,		/* Vty node. */
+	FPM_NODE,		/* Dataplane FPM node. */
+	LINK_PARAMS_NODE,       /* Link-parameters node */
+	BGP_EVPN_VNI_NODE,      /* BGP EVPN VNI */
+	/* RPKI node for configuration of RPKI cache server connections. */
+	RPKI_NODE,
+	BGP_FLOWSPECV4_NODE, /* BGP IPv4 FLOWSPEC Address-Family */
+	BGP_FLOWSPECV6_NODE, /* BGP IPv6 FLOWSPEC Address-Family */
+	BFD_NODE,	    /* BFD protocol mode. */
+	BFD_PEER_NODE,       /* BFD peer configuration mode. */
+	BFD_PROFILE_NODE,    /* BFD profile configuration mode. */
+	OPENFABRIC_NODE,     /* OpenFabric router configuration node */
+	VRRP_NODE,	   /* VRRP node */
+	BMP_NODE,	    /* BMP config under router bgp */
+	NODE_TYPE_MAX,       /* maximum */
 };
```

Comment:
What do you changed here ? it seems that there is nothing new. If there is a change, please change only the pertinent lines